optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/AcceptorConfig.java

```diff
@@ -193,18 +193,22 @@ public void setFadingUndoMoveTabuSize(Integer fadingUndoMoveTabuSize) {
         this.fadingUndoMoveTabuSize = fadingUndoMoveTabuSize;
     }
 
+    @Deprecated
     public Integer getSolutionTabuSize() {
```

Comment:
We should deprecate the fields too, so it's very clear, see TerminationConfig.calculateCountLimit